docs/cosa/run.md

```diff
@@ -257,6 +257,40 @@ $ cosa run -c --netboot tmp/pxelinux/pxelinux.0 -m 4096
 
 See [this section](https://docs.fedoraproject.org/en-US/fedora-coreos/live-booting/#_booting_via_pxe) of the official docs for more info.
 
+### PXE with Shim and UEFI Secure Boot
+
+1. Download a kernel, initramfs, rootfs and shim binary
+2. Combine initramfs + roots into a single file: `cat kernel.img initramfs.img > combined.img`
+3. Create an empty disk image (bug in kola): `touch disk.img`
```

Comment:
What did you hit?

Reply:
kola asks for a disk image:
```
$ cosa run -c --netboot pxe/shimx64.efi -m 4096 --qemu-firmware uefi-secure
COREOS_ASSEMBLER_ADD_CERTS=y
+cosa:16> podman run --rm -ti --security-opt 'label=disable' --privileged '--uidmap=1000:0:1' '--uidmap=0:1:1000' --uidmap 1001:1001:64536 -v /var/home/tim/projects/rhcos.images:/srv/ --device /dev/kvm --device /dev/fuse --tmpfs /tmp -v /var/tmp:/var/tmp '--volume=/etc/pki/ca-trust:/etc/pki/ca-trust:ro' quay.io/coreos-assembler/coreos-assembler:latest run -c --netboot pxe/shimx64.efi -m 4096 --qemu-firmware uefi-secure
Error: No disk image provided
2024-05-16T15:41:40Z cli: No disk image provided
```

Reply:
Not sure why, have not dig into this yet

Reply:
https://github.com/coreos/coreos-assembler/issues/3807 thanks to @jbtrystram